dockers/docker-fpm-frr/base_image_files/TSA

```diff
@@ -1,3 +1,14 @@
 #!/bin/bash
 
+# toggle the mux to standby if dualtor and any mux active
+[[ "$(sonic-cfggen -d -v DEVICE_METADATA.localhost.subtype | tr [:upper:] [:lower:])" == *"dualtor"* ]]
```

Comment:
Suggest change this and line 6 to if statements/blocks.

Reply:
Re-formatted the conditionals